zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/launcher/SparkInterpreterLauncher.java

```diff
@@ -77,6 +72,10 @@ public SparkInterpreterLauncher(ZeppelinConfiguration zConf, RecoveryStorage rec
         env.put(key, propValue);
       }
       if (isSparkConf(key, propValue)) {
+        if (Objects.equals("spark.driver.extraJavaOptions", key)) {
```

Comment:
It's better to add a comment explaining why handle `spark.driver.extraJavaOptions` in a specific way?

Reply:
thanks a lot for review  @zjffdu,  it's done.